test/system/run.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+
+load 'libs/bats-support/load'
+load 'libs/bats-assert/load'
+load 'libs/helpers'
+
+# It seems like 'toolbox run' (or 'enter') doesn't work fine when
+# the workdir is outside the $HOME.
+# This hack is to make the tests work from outside the $HOME. 
+readonly CURDIR=$PWD
+
+setup() {
+  cd "$HOME" || return 1
+  cleanup_containers
+}
+
+teardown() {
+  cleanup_containers
+  cd "$CURDIR" || return 1
+}
+
+
+@test "Echo 'Hello World' with no containers created (should fail)" {
```

Comment:
Try to stick to the "Try to" convention or create a different one if you don't like it.

Reply:
Actually, I tried to stick to the original tests. I only found 3 tests that were following that convention, so I wasn't sure what to do. When I changed a test's title, I tried to stick with the rest of the titles from that file.

Here is the difference between the current tests and my proposal:
```diff
--- 001-version.lst	
+++ version.lst	
 Output version number using full flag
+Output version number using command

--- 002-help.lst	
+++ help.lst	
 Show usage screen when no command is given
+Show usage screen when command help is given
+Show usage screen when flag --help is given

--- 101-list.lst	
+++ list.lst	
-Run list with zero containers and two images
+Run list with zero containers and images
 Run list with zero containers (-c flag)
 Run list with zero images (-i flag)
+Run list with zero toolbox's containers and images, but other image

--- 102-create.lst	
+++ create.lst	
 Create the default container
-Create a container with a valid custom name ('not-running'; name passed as an argument)
-Create a container with a custom image and name ('running';f29; name passed as an option)
+Create a container with a valid custom name ('not-running')
+Create a container with a custom image and name ('running';f29)
 Try to create a container with invalid custom name

--- 103-list.lst	
+++ list.lst	
-Run list with three containers and two images
+Run list with zero containers and images
+Run list with zero containers (-c flag)
+Run list with zero images (-i flag)
+Run list with zero toolbox's containers and images, but other image

--- 201-run.lst	
+++ run.lst	
+Echo 'Hello World' with no containers created (should fail)
 Echo 'Hello World' inside of the default container
 Echo 'Hello World' inside of the 'running' container
-Stop the 'running' container using 'podman stop'
 Echo 'Hello World' again in the 'running' container after being stopped and exit

--- 301-rm.lst	
+++ rm.lst	
 Try to remove a nonexistent container
 Try to remove the running container 'running'
 Remove the not running container 'not-running'
 Force remove the running container 'running'
-Force remove all remaining containers (only 1 should be left)
+Force remove all remaining containers (with 2 containers created and 1 running)

--- 302-rmi.lst	
+++ rmi.lst	
-Remove all images (2 should be present; --force should not be necessary)
+Remove all images with the default image present
+Fail to remove all images with the default image present and running
+Remove all images with the default image present and running (with flag --force)
```

Maybe it's a good time to convert all those tests to the convention or to define a new one.